.changeset/funny-foxes-allow.md

```
@@ -0,0 +1,5 @@
+---
+"blog": minor
+---
+
+Write Starlight Sidebar CSS adjustment guide
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)